data/sheragi/sheragi outfits.txt

```diff
@@ -149,12 +205,102 @@ outfit "Particle Waveform Turret"
 		"reload" 60
 		"firing energy" 180
 		"firing heat" 651
-		"shield damage" 110
+		"shield damage" 95
 		"hull damage" 270
-		"heat damage" 1344
+		"heat damage" 224
+		"ion damage" 5.6
 		"turret turn" 1.8
-	description "This ancient Sheragi turret accelerates a dense packet of electrons that, while very ineffective against any kind of shielding, has a very long range and easily melts through matter. By the nature of these weapons, one can deduce that their tactics likely involved using the Dragonflame Cannon at long range to break the enemy shields and finishing them off at relatively close range with the Particle Waveform Turrets."
+	description "This ancient Sheragi turret accelerates a dense packet of electrons that, while very ineffective against any kind of shielding, has a very long range and disrupts electrical systems. By the nature of these weapons, one can deduce that their tactics likely involved using the Dragonflame Cannon at long range to break the enemy shields and finishing them off at relatively close range with the Particle Waveform Turrets."
+
+outfit "Shard Fabricator"
+	category "Secondary Weapons"
+	cost 330000
+	thumbnail "outfit/shard"
+	"mass" 10
+	"outfit space" -10
+	"weapon capacity" -7
+	"fuel capacity" 36
+	"gun ports" -1
+	"operating costs" 150
+	weapon
+		sprite "projectile/shardactive"
+			"frame rate" 30
+			"random start frame"
+		sound "grab-strike"
+		"firing fuel" 2
+		icon "icon/shard"
+		"fire effect" "basic launch"
+		"hit effect" "Shard impact"
+		"submunition" "Shard inactive"
+		"inaccuracy" 80
+		"velocity" 15
+		"acceleration" 1.5
+		"drag" .1
+		"turn" 8
+		"lifetime" 40
+		"random lifetime" 20
+		"homing" 4
+		"radar tracking" .8
+		"reload" 60
+		"missile strength" 1
+		"burst count" 6
+		"burst reload" 2
+		"firing energy" 40
+		"firing heat" 60
+	description `To reduce reliance on supply chains during prolonged war time, this outfit contains specialized fabricators that produce very simple kinetic missiles from stock materials and a small amount of stored fuel; their simple guidance system merely receives data from the shipboard radar. The missiles are fired in volleys of six at a time from each launcher, making them very effective at overwhelming point defenses despite their fragility, but the low amount of fuel they carry means they can only home in on the target for about a second before they turn into unguided ammunition. Because they are manufactured on the spot, it takes a long time to reload.`
```

Comment:
```suggestion
	description `To reduce reliance on supply chains during prolonged war time, this outfit contains specialized fabricators that produce simple kinetic missiles from stock materials and a small amount of stored fuel; their guidance system merely receives data from the shipboard radar. The missiles are fired in volleys of six at a time from each launcher, making them very effective at overwhelming point defenses despite their fragility. However, the low amount of fuel they carry means they can only home in on the target for about a second before they turn into unguided ammunition. Because they are manufactured on the spot, it takes a long time to reload.`
```
Trying to remove unnecessary/repetitive words. That second-to-last sentence was especially long (and thus a little draining to read), and a period would help give the reader a break.